static/app/components/textCopyInput.tsx

```diff
@@ -33,17 +32,11 @@ function TextCopyInput({
   children,
   ...inputProps
 }: Props) {
-  const textRef = useRef<HTMLInputElement>(null);
+  const textNodeId = useId();
 
   const handleSelectText = useCallback(() => {
-    if (!textRef.current) {
-      return;
-    }
-
-    // We use findDOMNode here because `this.textRef` is not a dom node,
-    // it's a ref to AutoSelectText
-    const node = findDOMNode(textRef.current); // eslint-disable-line react/no-find-dom-node
-    if (!node || !(node instanceof HTMLElement)) {
+    const node = document.getElementById(textNodeId);
```

Comment:
`getElementById` might meant that the `else` case below isn't needed anymore... it should always be an HTMLInputElement .... unless the `<StyledInput>` implementation changes...